adminer/drivers/oracle.inc.php

```diff
@@ -165,7 +165,7 @@ function connect() {
 	}
 
 	function get_databases() {
-		return get_vals("SELECT tablespace_name FROM user_tablespaces");
+		return get_vals("SELECT distinct tablespace_name FROM all_tables where tablespace_name is not null");
```

Comment:
Why `DISTINCT`? What are the other columns of this table in rows with the same `tablespace_name`? Shouldn't you add a condition instead?